nuxeo-core/nuxeo-core/src/main/resources/OSGI-INF/coldstorage-contrib.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.core.event.coldstorage.contribution">
+
+  <!-- define the event that will be fired by the scheduler -->
+  <extension target="org.nuxeo.ecm.core.scheduler.SchedulerService" point="schedule">
+    <schedule id="checkAvailabilityOfBlobsBeingRetrieved">
+      <!-- every hour and 7 minutes -->
+      <cronExpression>0 7 * ? * * *</cronExpression>
+      <event>checkAvailabilityOfColdStorageContentBeingRetrieved</event>
+    </schedule>
+  </extension>
+
+  <!-- define the listener that will handle the scheduler event -->
+  <extension target="org.nuxeo.ecm.core.event.EventServiceComponent" point="listener">
+    <listener name="checkAvailabilityOfBlobsBeingRetrieved" async="true"
```

Comment:
Since the listener is implementing `EventListener`, it is registered as synchronous (this is historical and unclear code...).
So you should remove `async="true"` (it's false by default).

Reply:
ah ok then i will do the modification